docs/EN/Installing-AndroidAPS/Update-to-new-version.md

```diff
@@ -9,7 +9,11 @@
 
    ![Git installation path](../images/Update_GitPath.png)
 
-* Let Studio know where is git.exe located: File - Settings 
+  On a Mac use homebrew to install git: ```$ brew install git```.
+  
+  For details on installing git see the [official git documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+* Let Studio know where is git.exe located: File - Settings <i>(Mac: File - Preferences)</i>
```

Comment:
On a Mac the preferences are not below "file" but the app name.
Nevertheless: if installed vie Homebrew, no need to change any preferences.

Reply:
Corrected. Also separated Win & Mac with own sub-headlines for better reading.